apps/hash-ai-worker-py/app/workflows.py

```diff
@@ -0,0 +1,17 @@
+from datetime import timedelta
+from temporalio import workflow
+
+with workflow.unsafe.imports_passed_through():
```

Comment:
Do they say to use `unsafe` in the docs? if so.. pretty strange naming

Reply:
Yes it seems so.
https://github.com/temporalio/sdk-python/blob/main/README.md#implementing-a-workflow
I believe the naming is like this to make it clear that you're passing through their "sandbox" for any imports defined underneath